extensions/amp-user-location/validator-amp-user-location.protoascii

```diff
@@ -33,6 +33,13 @@ tags: {  # <amp-user-location>
   amp_layout: {
     supported_layouts: NODISPLAY
   }
+  attrs: {
+    name: "src"
+    value_url: {
+      protocol: "https"
+      allow_relative: true
```

Comment:
If you mandate `https`, you should set `allow_relative` to false in new rules. The reason is that the protocol check is unaware of the document URL. So, you are effectively allowing `http` by allowing relative URLs.

Reply:
Hmm I see, that makes sense. I'll make a followup PR.